mteb/models/openai_models.py

```diff
@@ -15,27 +15,49 @@
 
 
 class OpenAIWrapper(Wrapper):
-    def __init__(self, model_name: str, embed_dim: int | None = None, **kwargs) -> None:
+    def __init__(
+        self,
+        model_name: str,
+        max_seq_length: int,
```

Comment:
You need to update `ModelMetadata`s to pass `max_seq_length` and I think it will be good to pass `tokenizer` as well